Packs/ExpanseV2/Integrations/ExpanseV2/ExpanseV2.py

```diff
@@ -2435,6 +2573,7 @@ def main() -> None:
             business_units = argToList(params.get('business_unit'))
             issue_types = argToList(params.get('issue_type'))
             tags = argToList(params.get('tag'))
+            cloud_management_status = argToList(params.get('cloud_management_status'))
```

Comment:
```suggestion
            cloud_management_status = params.get('cloud_management_status')
```

You call argToList to this argument again inside `fetch_incidents` method